csharp/src/Ice/OutgoingRequest.cs

```diff
@@ -161,15 +161,16 @@ public sealed class OutgoingRequestWithStructParam<TParamList, TReturnValue>
     // publicly visible Ice-internal class used as base class for void requests
     public class OutgoingRequest
     {
-        private protected void Invoke(IObjectPrx prx, OutgoingRequestFrame request)
+        private protected void Invoke(IObjectPrx prx, OutgoingRequestFrame request, bool oneway)
```

Comment:
I thought so as well, but we also could do the same for `_operationName` and `_idempotent`, and go out of our way not to, so I just followed suit. Should I also relocate these fields as well then?

Reply:
I just would keep these 2 fields where they are right now, since they are used by the derived classes in calls to `WithEmpytParamList` and `WithParamList`. We may improve this again in a future PR.

Reply:
Alright, I'll ignore them for now, but I definitely think it could be improved in the future!